### ISSUE TITLE
[Suggestion/Question] Space-brakes/Parking brake 

### ISSUE THREAD

**Issue by TIKIRobo** [tsunamayo/Starship-EVO#2743]
So this may seem a bit weird but I think it would be cool to see something similar to the inertia dampeners like in Space Engineers.

Let's say you have a very nice space station but rude AI/Players keep on ramming into the station making it move around, wouldn't it be great if you could push a button and the station's thrusters would try to keep it stable? Or what if you have a hover craft that you would love to leave hovering but also for it to not slide down a hill? Also if you wanted to build some sort of flying base on the ring planets but it would just fall as soon as you left your seat wouldn't it be cool if you could make it stay hovering there so you could walk around on it or maybe even land another ship on it.

Also maybe i'm dumb and you can already do this but it would be cool if turrets that auto fire and auto target could work even if you aren't flying the ship so that way you could set up hovering turrets around your hovering base or what ever you want.

**Comment by SovietSponge** [comment]
Some way to make stations we build properly stationary would be nice.

**Comment by tsunamayo** [comment]
Yes I want to give a stations anchor block so it stays static, and a station core to control your turret even when nobody is flying them (or something else).
